### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by nickhsine**
### Bug fix
- bug fix: render bookmark widget on aside:mobile
- bug fix: link of donation box(generated by SSR) does not contain utm query param

### CODE REVIEW

File: src/components/donation-box.js [hunk 1]
```diff
@@ -78,34 +77,41 @@ const Donate = styled.div`
...
         <Text>{_content.desc}</Text>
         <Donate>
-          <ReactGA.OutboundLink
-            eventLabel={`[article_donation_button_click]: ${currentHref}`}
-            to={donateURL || _donatePath}
-            target="_blank"
-          >
-            <p>{_content.bt}</p>
-          </ReactGA.OutboundLink>
+          {isMounted ? (
```
Comment (YuCJ) [inline src/components/donation-box.js]: Rendering the element when the component is not mounted seems okay. It just take the original `donateURL` without `utm`.

**Review by YuCJ — COMMENTED** [review]
LGTM 👍

**Comment by nickhsine** [comment]
@YuCJ 
Updated.
About donation link change, please see the https://github.com/twreporter/react-article-components/pull/40/commits/537089416bb39ab7956b4982063899c31d25fb24 .
This change is based on [this PR](https://github.com/twreporter/twreporter-npm-packages/pull/11).

**Comment by nickhsine** [comment]
rebase commits

**Pull request merged**